src/connectivity.jl

```diff
@@ -285,13 +285,14 @@ from `v`.
 - `dir=:out`: If `g` is directed, this argument specifies the edge direction
 with respect to `v` of the edges to be considered. Possible values: `:in` or `:out`.
 """
-function neighborhood(g::AbstractGraph, v::Integer, d::Integer; dir=:out)
+neighborhood(g::AbstractGraph, v::Integer, d::Integer; dir=:out) =
```

Comment:
Could be, but perfection being the enemy of good at this point, I'd like to proceed with this (since we know it's performant) and figure out whether or not we can simplify it later.